src/testing/docker/default-docker-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -ex
+#### build image
+docker build -t fossology/fossology-test . 
+
+#### create network
+docker network create --subnet=172.18.0.0/16 fossology-testnet
+
+#### create container and set IP
+docker create -p 8081:8081 --name fossology-test --net fossology-testnet --ip 172.18.0.22 fossology/fossology-test
+
+#### start container
+docker start fossology-test
+
+#### is container running
+docker ps | grep -q fossology/test
```

Comment:
Are you sure that `fossology/test` is correct. Maybe it has to be `fossology-test`.